src/com/cdd/bao/template/Vocabulary.java

```diff
@@ -418,66 +436,69 @@ private void loadLabels(File baseDir, File[] extra, Set<String> exclude) throws
-			if (eliminatedTerms.contains(subject.getURI())) continue;
+			if (eliminatedTerms.contains(subjURI)) continue;
 
 			// separately collect everything that's part of the class/property hierarchy
 			if (predicate.equals(subPropOf) && object.isURIResource())
 			{
-				anyProp.add(object.asResource().getURI());
-				anyProp.add(subject.getURI());
+				anyProp.add(objURI);
+				anyProp.add(subjURI);
 			}
 			else if (predicate.equals(subClassOf) && object.isURIResource())
 			{
-				anyValue.add(object.asResource().getURI());
-				anyValue.add(subject.getURI());
+				anyValue.add(objURI);
+				anyValue.add(subjURI);
 			}
 			else if (predicate.equals(rdfType) && (object.equals(owlDataType) || object.equals(owlObjProp)))
 			{
-				anyProp.add(subject.getURI());
+				anyProp.add(subjURI);
 			}
 			
 			if (!object.isLiteral()) continue;
 
-			String uri = subject.getURI();
 			String label = object.asLiteral().getString();
 
 			if (predicate.equals(propLabel))
 			{
-				uriToLabel.put(uri, label);
+				uriToLabel.put(subjURI, label);
 				String[] list = labelToURI.get(label);
-				labelToURI.put(label, ArrayUtils.add(list, uri));
+				labelToURI.put(label, ArrayUtils.add(list, subjURI));
 			}
 			else if (predicate.equals(propDescr))
 			{
-				uriToDescr.put(uri, label);
+				uriToDescr.put(subjURI, label);
 			}
 			else if (predicate.equals(pubchemSource))
 			{
-				uriToPubChemSource.put(uri, label);
+				uriToPubChemSource.put(subjURI, label);
 			}
 			else if (predicate.equals(pubchemImport))
 			{
-				uriToPubChemImport.put(uri, object.asLiteral().getBoolean());
+				uriToPubChemImport.put(subjURI, object.asLiteral().getBoolean());
 			}
 			else if (predicate.equals(externalURL))
 			{
-				String[] list = uriToExternalURLs.get(uri);
-				uriToExternalURLs.put(uri, ArrayUtils.add(list, label));
+				String[] list = uriToExternalURLs.get(subjURI);
+				uriToExternalURLs.put(subjURI, ArrayUtils.add(list, label));
 			}
 			else if (predicate.equals(altLabel1) || predicate.equals(altLabel2) || predicate.equals(altLabel3) || predicate.equals(altLabel4))
 			{
-				String[] list = uriToAlternateLabels.get(uri);
-				if (!ArrayUtils.contains(list, label)) uriToAlternateLabels.put(uri, ArrayUtils.add(list, label));
+				String[] list = uriToAlternateLabels.get(subjURI);
+				if (!ArrayUtils.contains(list, label)) uriToAlternateLabels.put(subjURI, ArrayUtils.add(list, label));
 			}
 		}
-		
+
 		// pull out the equivalences
 		Property owlEquivalence = model.createProperty(ModelSchema.PFX_OWL + "equivalentClass");
 		for (StmtIterator iter = model.listStatements(null, owlEquivalence, (RDFNode)null); iter.hasNext();)
 		{
 			Statement stmt = iter.next();
-			String uri1 = stmt.getSubject().getURI(), uri2 = stmt.getObject().asResource().getURI();
+			String uri1 = remapIfAny(stmt.getSubject().getURI());
+			String uri2 = remapIfAny(stmt.getObject().asResource().getURI());
```

Comment:
You could rename `uri1` and `uri2` to `subjURI` and `objURI` to be consistent with the code above. This also applies to code further down.

Reply:
Implementing these changes shortly.